apps-client/mobile/components/poll/poll-card.tsx

```diff
@@ -0,0 +1,353 @@
+import React from 'react'
+import { View } from 'react-native'
+
+import { Badge } from '@pple-today/ui/badge'
+import { Button } from '@pple-today/ui/button'
+import { Icon } from '@pple-today/ui/icon'
+import { cn } from '@pple-today/ui/lib/utils'
+import { Text } from '@pple-today/ui/text'
+import dayjs from 'dayjs'
+import duration from 'dayjs/plugin/duration'
+import { useRouter } from 'expo-router'
+import { ChevronRightIcon, Clock3Icon, MessageCircleQuestionIcon } from 'lucide-react-native'
+
+import type { FeedItemPoll } from '@api/backoffice/app'
+import { exhaustiveGuard } from '@app/libs/exhaustive-guard'
+
+import { PollOptionItem, PollOptionResult } from './poll-option'
+
+type PollItem = FeedItemPoll['poll']
+
+dayjs.extend(duration)
+
+export const PollCardContent = (props: { feedItem: FeedItemPoll }) => {
+  const router = useRouter()
+  const [isEnded, setIsEnded] = React.useState(dayjs().isAfter(dayjs(props.feedItem.poll.endAt)))
+
+  const renderSeeMoreOptions = () => {
+    if (props.feedItem.poll.options.length > 3) {
+      return (
+        <Button
+          variant="secondary"
+          className="w-full text-start border-2 border-base-outline-default flex flex-row justify-between items-center bg-base-bg-white font-body-medium h-11 rounded-2xl mt-2"
+          onPress={() => {
+            router.navigate(`./feed/${props.feedItem.id}`)
+          }}
+        >
+          <Text className="font-body-medium text-base-text-medium text-sm">
+            {`ดูเพิ่มเติม (อีก ${props.feedItem.poll.options.length - 3} ตัวเลือก)`}
+          </Text>
+          <Icon icon={ChevronRightIcon} size={24} strokeWidth={1} />
+        </Button>
+      )
+    }
+
+    return null
+  }
+
+  return (
+    <View className="gap-3">
+      <View className="py-[13px] px-2 flex flex-col bg-base-bg-default rounded-xl mx-4">
+        <PollHeader
+          isEnded={isEnded}
+          triggerEnded={() => setIsEnded(true)}
+          poll={props.feedItem.poll}
+        />
+        {isEnded ? (
+          <PollOptionResultList poll={props.feedItem.poll} card />
+        ) : (
+          getPollCardOptionList(props.feedItem.poll)
+        )}
+        {renderSeeMoreOptions()}
+      </View>
+    </View>
+  )
+}
+
+export const PollDetailContent = (props: { feedItem: FeedItemPoll }) => {
```

Comment:
I think if `PollDetailContent` is just `PollCardContent` without the renderSeeMoreOptions
you can merge the two together and add another props